### PR TITLE
Publish interview with Devney Hamilton of Sassafras

### PR DESCRIPTION
**Pull request by anaulin**

### CODE REVIEW

File: content/episode/devney-hamilton-sassafras-tech.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+* Website: [sassafras.coop](https://www.sassafras.coop/)
+* Twitter: [@sassafrastech](https://twitter.com/sassafrastech)
+* Blog: [blog.sassafras.coop](https://blog.sassafras.coop/)
+
+Organizations mentioned:
+* [Atomic Object](https://atomicobject.com/), a Midwest software consultancy, employee-owned B corporation
+* [Loomio](https://help.loomio.org/en/about/), a software product co-op.
+* [Allied Media Conference](https://amc.alliedmedia.org/)
+* [INCITE! Women of Color Against Violence](https://incite-national.org/)
+* [Design Action Collective](https://designaction.org/)
```
Comment (DevneyHamilton) [inline content/episode/devney-hamilton-sassafras-tech.md]: Thanks for including these! Could we please swap the positions of Atomic Object and Design Action Collective?

File: content/episode/devney-hamilton-sassafras-tech.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
++++
+
+## Episode Notes
+
+Sassafras encourages people of color and/or with disabilities and/or trans or gender-non-conforming, or otherwise targeted with oppression in tech to contact them at [info@sassafras.coop](mailto:info@sassafras.coop).
+
+Ways to stay in touch with Sassafras:
+* Website: [sassafras.coop](https://www.sassafras.coop/)
+* Twitter: [@sassafrastech](https://twitter.com/sassafrastech)
+* Blog: [blog.sassafras.coop](https://blog.sassafras.coop/)
```
Comment (DevneyHamilton) [inline content/episode/devney-hamilton-sassafras-tech.md:18]: Thanks! For now could we please swap the positions of the Website and Blog? The Blog is most up-to-date for the foreseeable future :)

File: content/episode/devney-hamilton-sassafras-tech.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+Organizations that help fund cooperatives:
+* [The Working World](https://www.theworkingworld.org/)
+* [Seed Commons](https://seedcommons.org/)
+
+Organizations that provide resources for starting coops:
+* [US Federation of Worker Coops](https://www.usworker.coop/)
+* [Sustainable Economies Law Center](https://www.theselc.org/)
+* [Democracy at Work Institute](https://institute.coop/)
+
+Books and reading recommended:
```
Comment (DevneyHamilton) [inline content/episode/devney-hamilton-sassafras-tech.md:49]: perfect- thanks for pulling these out for the audience :D

File: content/episode/devney-hamilton-sassafras-tech.md [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+* [Prefigurative Design as a Method for Research Justice](https://dl.acm.org/doi/pdf/10.1145/3359302), research paper by Mariam Asad
+* [Collective Courage: A History of African American Cooperative Economic Thought and Practice](https://www.psupress.org/books/titles/978-0-271-06216-7.html), book by Jessica Gordon Nembhard
+* [Emergent Strategy: Shaping Change, Changing Worlds](https://www.akpress.org/emergentstrategy.html), book by adrienne maree brown
+
+## Transcript
+
```
Comment (DevneyHamilton) [inline content/episode/devney-hamilton-sassafras-tech.md:55]: The launch.coop conversation is all set to keep included :) thanks for double checking!

File: content/episode/devney-hamilton-sassafras-tech.md [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+{{< timestamp "[00:45:13]" >}} _Ana:_ How old is Sassafras?
+
+{{< timestamp "[00:45:15]" >}} _Devney:_ That's a great question. I think we have 2013 as our starting time. I should check our bylaws and see when the earliest version of the like confirmed bylaws. Yeah, eight years.
+
+{{< timestamp "[00:45:29]" >}} _Ana:_ That's a very respectable amount of years for a business.
+
+{{< timestamp "[00:45:34]" >}} _Devney:_ Around that time, Design Action Collective, in the Bay Area was a little older than that. And I know they were a big inspiration for when Sassafras was pretty, still kinda young. It was like, okay, well, Design Action has been at it so this might actually work. So many thanks to them.
+
+{{< timestamp "[00:45:56]" >}} There's also a lot of history in Collective Courage, which I would recommend people check out. Collective Courage, a History of African-American Cooperative Economic Thought and Practice, by Jessica Gordon Nembhard. So having that history is also really grounding for us. People have been at this for a very long time. It is a way to build power. I recommend people check it out.
```
Comment (jpdimond) [inline content/episode/devney-hamilton-sassafras-tech.md:340]: Dr. Jessica Gordon Nembhard, PhD

Comment (anaulin) [reply content/episode/devney-hamilton-sassafras-tech.md:340]: Fixed in episode notes section.

File: content/episode/devney-hamilton-sassafras-tech.md (hunk 2 above)
Comment (anaulin) [reply content/episode/devney-hamilton-sassafras-tech.md:18]: Done.

File: content/episode/devney-hamilton-sassafras-tech.md (hunk 1 above)
Comment (anaulin) [reply content/episode/devney-hamilton-sassafras-tech.md]: Done.

**Pull request merged**